01-introduction.Rmd

```diff
@@ -1,6 +1,5 @@
 # Introduction {#intro}
 
-<!-- to do things sounds rather vague without context-->
 This book is about harnessing the power of modern computers to make the most of geographic data.
```

Comment:
More verbose than current wording. Sounds like hyperbole to me, which I'm sometimes guilty of!